cli/dump_test.go

```diff
@@ -1,58 +0,0 @@
-package main
-
-import (
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"testing"
-
-	"github.com/nspcc-dev/neo-go/pkg/config"
-	"github.com/stretchr/testify/require"
-	"gopkg.in/yaml.v2"
-)
-
-func TestDBRestore(t *testing.T) {
```

Comment:
Why moving it?

Reply:
The file was just renamed to include not only restore tests but also tests for the whole `cli/server/server.go`. Fixed, so now there're both `dump_test.go` and `server_test.go`.

Reply:
We can have both, not a problem.